src/components/Property/Tenure.js

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types'
 
 const Tenure = ({ tenure }) => {
   let tenureToShow = (tenure) => {
-    if (tenure?.typeCode) {
+    if (tenure.typeDescription) {
```

Comment:
Will this blow up if `tenure` is `{}` ? Safe navigator  in ruby `&.` has the js equivalent of `?.`

Reply:
I checked with` {}` it works fine 
I checked in console, when you do `tenure.typeDescription` it returns `false` if we have empty object 
I guess that's the way to check it in js

Reply:
Interesting... I just opened my console and tried `{}.bla` and you get an error. I wonder what's going on here that's different

Reply:
Ah you have to declare the object first. So it would return `undefined` and that interprets it as false